src/case_gather/views.py

```diff
@@ -11,7 +11,13 @@
 @api_view(['GET', 'POST'])
 @permission_classes((IsAuthenticatedOrReadOnly,))
 def case_list(request):
+    '''
+    get:
+        Get a list of all cases.
 
+    post:
```

Comment:
@AriHrannar / @Jarngreipr  should the `case_gather` even have a `POST` endpoint?